### PR TITLE
feat(Icon): [SPB-2461] add transfer icon

### PR DESCRIPTION
**Pull request by vshyrokov**
[SPB-2461](https://toptal-core.atlassian.net/browse/SPB-2461)

### Description

Add transfer icon

links

[Transfer16](https://app.abstract.com/projects/1b06c884-06af-482a-bf12-a82f521a19a1/branches/master/commits/11bd96b68a1a36280cf153a95252bbcec2381b17/files/2fbda9fd-63f3-42e7-a9a8-135a6562ac01/layers/BF8F6B07-5CEF-46D2-BCA1-B016BDA8827E)
[Transfer24](https://app.abstract.com/projects/1b06c884-06af-482a-bf12-a82f521a19a1/branches/master/commits/11bd96b68a1a36280cf153a95252bbcec2381b17/files/2fbda9fd-63f3-42e7-a9a8-135a6562ac01/layers/3B329A1F-B061-4E9A-9AE1-1160896AE0F2)

### Screenshots

image

### Review

- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/api-principles.md)
- `N/A` Annotate all `props` in component with documentation
- `N/A` Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure that tests pass by running `yarn test`
- [x] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

### CODE REVIEW

File: packages/picasso-charts/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "peerDependencies": {
     "@material-ui/core": "4.11.0",
     "@toptal/picasso": "^6",
-    "@toptal/picasso-shared": "5.0.1",
+    "@toptal/picasso-shared": "5.0.2",
```
Comment (vshyrokov) [inline packages/picasso-charts/package.json:27]: for some reason it didn't allow me to commit without this sync, that was done with ` yarn syncpack:fix`

**Comment by vshyrokov** [comment]
ping review

**Comment by vshyrokov** [comment]
@toptal-anvil ping reviewers

**Review by teimurjan — APPROVED** [review]

**Comment by vshyrokov** [comment]
@toptal-bot run all

**Review by OleksandrNechai — APPROVED** [review]

**Pull request merged**